bookkeeper-server/src/main/java/org/apache/bookkeeper/util/EventLoopUtil.java

```diff
@@ -47,6 +47,10 @@ public static EventLoopGroup getServerEventLoopGroup(ServerConfiguration conf, T
         return getEventLoopGroup(threadFactory, conf.getServerNumIOThreads(), conf.isBusyWaitEnabled());
     }
 
+    public static EventLoopGroup getServerAcceptorGroup(ServerConfiguration conf, ThreadFactory threadFactory) {
+        return getEventLoopGroup(threadFactory, conf.getServerNumAcceptorThreads(), conf.isBusyWaitEnabled());
```

Comment:
does BusyWaitEnabled apply to the acceptor  group ?

Reply:
I changed it to false. Thanks~